tide/master/ui/MasterQuickView.h

```diff
@@ -58,7 +58,8 @@ class MasterQuickView : public QQuickView
 
 public:
     /** Constructor. */
-    MasterQuickView(OptionsPtr options, const MasterConfiguration& config);
+    MasterQuickView(OptionsPtr options, const MasterConfiguration& config,
+                    LockPtr lock);
```

Comment:
For readability group options and lock parameters, leaving MasterConfiguration at the end.